packages/angular/cli/lib/config/schema.json

```diff
@@ -398,6 +401,33 @@
         "^[a-z]{1,3}-.*": {}
       },
       "definitions": {
+        "i18n": {
+          "description": "Project i18n options",
+          "type": "object",
+          "properties": {
+            "missingTranslation": {
```

Comment:
@clydin, should `missingTranslation` be at this level, builder level or both?

Reply:
This seems more of a build time behavior than a description of the translations for the project.

This option actually brings up a bigger issue in that the CLI currently does not have a mechanism to control warnings.  A mechanism to enable/disable warnings and treat warnings as errors would be useful in general.  This would then be one of those warnings.

Reply:
Do you prefer a new option for `missingTranslation ` or use the existing `i18nMissingTranslation`?

Reply:
might as well use the existing since it simplifies any deprecation logic and migrations.